ci-operator/step-registry/azure/bastion/pre/azure-bastion-pre-commands.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+EOF
```

Comment:
The test show `INFO[2022-01-12T20:51:38Z] /bin/bash: line 8: EOF: command not found`